rosbag2_transport/src/rosbag2_transport/qos.cpp

```diff
@@ -19,20 +19,37 @@
 
 #include "qos.hpp"
 
+namespace
+{
+// These values are returned to mean "unspecified" in FastDDS
```

Comment:
Thanks for this comment explaining the fix, maybe include it in the description?